price/domain/price.go

```diff
@@ -325,14 +325,14 @@ func (p Price) GetPayableByRoundingMode(mode string, precision int) Price {
 	if p.IsNegative() {
 		negative = -1
 	}
-	offsetToCheckRounding := new(big.Float).Mul(p.precisionF(precision), new(big.Float).SetInt64(10))
+	offsetToCheckRounding := new(big.Float).Mul(p.precisionF(precision), new(big.Float).SetInt64(100))
```

Comment:
@danielpoe is it possible that we have in fact a too less precision on rounding?

I modified the GetPayable test and added a call for 88.39499 and got a Payable of 88.39 - but imho we should get a Payable of 88.40 for that value. 

Thus I increased the rounding precisions here and adapted the rounding up/down comparisons to 
- round up if we get xx.xx45 for halfup
- round down if we get xx.xx55 for halfdown

But I am also pretty unsure about this now...